cli-test/tests/index.js

```diff
@@ -80,22 +80,29 @@ test(`prettier-eslint ${writeCommand}`, async () => {
   // we have to recreate and delete the files every time
   const example1Path = path.resolve(__dirname, '../fixtures/example1.js')
   const example2Path = path.resolve(__dirname, '../fixtures/example2.js')
-  const example1 = `const {  example1  }  =  baz.bar`
-  const example2 = `function example2(thing){return thing;};;;;;;;;;`
-  await Promise.all([
-    pWriteFile(example1Path, example1),
-    pWriteFile(example2Path, example2),
-  ])
-  const stdout = await runPrettierESLintCLI(writeCommand)
-  expect(stdout).toMatchSnapshot(`stdout: prettier-eslint ${writeCommand}`)
-  const [example1Result, example2Result] = await Promise.all([
-    pReadFile(example1Path, 'utf8'),
-    pReadFile(example2Path, 'utf8'),
-  ])
-  expect({example1Result, example2Result}).toMatchSnapshot(
-    `file contents: prettier-eslint ${writeCommand}`,
-  )
-  await Promise.all([pUnlink(example1Path), pUnlink(example2Path)])
+  try {
```

Comment:
Most of this is just reformatted to make it so if the test fails, the files are still removed.